benchmarks/perception/a1_perception_2nodes/src/image_input_component.cpp

```diff
@@ -39,8 +39,12 @@ namespace perception
 ImageInputComponent::ImageInputComponent(const rclcpp::NodeOptions & options)
 : rclcpp::Node("ImageInputComponent", options)
 {
+
+  // Get the input_topic_name parameter from the parameter server with default value "input"
+  input_topic_name = this->declare_parameter<std::string>("input_topic_name", "input");
+
   // Create image pub
-  pub_image_ = image_transport::create_camera_publisher(this, "input");
+  pub_image_ = image_transport::create_camera_publisher(this, input_topic_name);
```

Comment:
Is `input_topic_name` used anywhere else besides the node constructor?  If not, consider making `input_topic_name` locally scoped within the constructor as there is no need for a (protected) member.

Reply:
gotcha! No, it's not used anywhere else. I changed it to be locally scoped. Thanks!